dao layer implement

Naveenkumar-ramesh opened the pull request.

ponvel-i2i commented on com/ideas2it/employee/dao/EmployeeDao.java:
```diff
@@ -0,0 +1,53 @@
+     * @return Return the boolean value.
+     */
+    @Override
+    boolean addEmployee(Employee employee);
```
Why do you add @Override annotation here?

ponvel-i2i commented on com/ideas2it/employee/mapper/ModelMapper.java:
```diff
@@ -0,0 +1,100 @@
+ * 
+ * @version 1.8 15-09-2022
+ * @author Naveenkumar R
+ */
```
Its not a model mapper. Its a EmployeeMapper

ponvel-i2i commented on com/ideas2it/employee/service/EmployeeManagementService.java:
```diff
@@ -1,25 +1,85 @@
  * @author Naveenkumar R
  */
-public interface EmployeeManagementService {
+public class EmployeeManagementService {
```
Why do you remove service interface

ponvel-i2i commented on com/ideas2it/employee/model/EmployeeDTO.java:
```diff
@@ -0,0 +1,117 @@
+package com.ideas2it.employee.model;
+
```
DTO package should be dto

Naveenkumar-ramesh commented:
Bro , when i moved comments from implement to interface i mistakenly moved
it too.
then when i compiled ,found the mistake and corrected it.

On Fri, Sep 16, 2022 at 9:49 AM Ponvel ***@***.***> wrote:

> ***@***.**** commented on this pull request.
> ------------------------------
>
> In com/ideas2it/employee/dao/EmployeeDao.java
> <https://github.com/Naveenkumar-ramesh/EmployeeManagementSystem/pull/2#discussion_r972599016>
> :
>
> > + *
> + * @version 1.8 13-09-2022
> + * @author Naveenkumar R
> + */
> +public interface EmployeeDao {
> +
> +
> +    /**
> +     * Saves the employee details in above database
> +     * and return true if the process is successful.
> +     *
> +     * @param employee from controller
> +     * @return Return the boolean value.
> +     */
> +    @Override
> +    boolean addEmployee(Employee employee);
>
> Why do you add @OverRide <https://github.com/OverRide> annotation here?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Naveenkumar-ramesh/EmployeeManagementSystem/pull/2#pullrequestreview-1110073635>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A3AXQHSLX5I7XCD2JZFNR7DV6PYMNANCNFSM6AAAAAAQNTBS7M>
> .
> You are receiving this because you authored the thread.Message ID:
> <Naveenkumar-ramesh/EmployeeManagementSystem/pull/2/review/1110073635@
> github.com>
>
